specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -1033,6 +1033,20 @@
         }
       },
       "description": "The minimum number of violations required within the selected lookback time window required to raise an alert."
+    },
+    "ErrorResponse": {
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=5&length=13)

Can you get this from common types instead?

Reply:
@grchris-microsoft Can you reply the question above?

Reply:
I tried to fix the errors that appear in this PR, but every fix led to many other errors. Therefore, I did some fixes, and some other which are more complicated, I added to suppression. 
The current version in Main is broken, and fixing it is definitely out of the current PR's scope